Lombiq.Tests.UI/Services/Counters/CounterProbe.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Text;
+
+namespace Lombiq.Tests.UI.Services.Counters;
+
+public abstract class CounterProbe : CounterProbeBase, IDisposable
```

Comment:
After a page change, write all counters' values to the TestOutput for debugging.